filebeat/module/traefik/access/ingest/pipeline.json

```diff
@@ -3,9 +3,9 @@
     "processors": [
         {
             "grok": {
-                "field": "traefik.access.message",
+                "field": "message",
                 "patterns": [
-                    "(?:%{NUMBER:http.response.body.bytes:long}|-)( (?:\"%{DATA:http.request.referrer}\"|-)?( (?:\"%{DATA:traefik.access.agent}\"|-)?)?( (?:%{NUMBER:traefik.access.request_count:long}|-)?)?( (?:\"%{DATA:traefik.access.frontend_name}\"|-)?)?( \"%{DATA:traefik.access.backend_url}\")?( %{NUMBER:temp.duration:long}ms)?)?"
+                    "%{IPORHOST:source.address} %{GREEDYDATA:traefik.access.user_identifier} %{GREEDYDATA:user.name} \\[%{HTTPDATE:traefik.access.time}\\] \"%{WORD:http.request.method} %{DATA:url.original} HTTP/%{NUMBER:http.version}\" %{NUMBER:http.response.status_code:long} (?:%{NUMBER:http.response.body.bytes:long}|-)?( (?:\"%{DATA:http.request.referrer}\"|-)? (?:\"%{DATA:traefik.access.agent}\"|-)? (?:%{NUMBER:traefik.access.request_count:long}|-)? (?:\"%{DATA:traefik.access.frontend_name}\"|-)? (?:\"%{DATA:traefik.access.backend_url}\"|-)?( %{NUMBER:temp.duration:long}ms|-)?)?"
```

Comment:
We should use the dissect processor here as we did in Filebeat for the first part: https://www.elastic.co/guide/en/elasticsearch/reference/master/dissect-processor.html It's more efficient then grok.

Reply:
Ok, fixed adding a dissect processor at the top of the pipeline